action.yml

```diff
@@ -19,14 +23,16 @@ runs:
       run: |
         rm -rf .terraform*
         VERSION=0.9 /bin/bash -c "$(curl -fsSL https://docs.runx.dev/install.sh)"
+        CONF=${CONF:-'opta.yml'}
```

Comment:
You can specify a default on line 15: https://docs.github.com/en/actions/creating-actions/metadata-syntax-for-github-actions#inputsinput_iddefault